domains/hudson.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Hudson-Liu",
+           "email": "",
+           "discord": "879464844103934014",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IOrkBDTOL8e5YwmSg1zQx2TOp0s6X984MjKxTc5Mwh9FajQFzQ6PkA_BfRn4LVFwuuVsURcPR-7J5Y4ZR0I35Ug1j1JCxjUHcgRyxMyiiiNnrQCGe2bdTDFKBaPbyIOKhUT94mv8YOh8pqQJVPPcEzQ9oXonSRsAMGGqE-4kAr41UIusklBWaA9F8Z8cRLEtIB1si9bMA6R3iLlTxEVYtXl7OgrRxF0Bt1gb9_fnF3oKTWT3w5S19XQYqfMD1RCwl3AKJEBpDq7Vcr0sgYlOYG-W_hWWMyxf7NNrIG-9Vz2UyXeM531eqP40tp1lIp7EWwE0TqdYcrRCaAafFS1D3g.TMk5oUDQEorIXe7kJK1gvA.N43xlqEWY9K4lmLeHC-5Z2C4BxkzdwLBKcO_p770ZdSWJbagO8mBzwZsFfvrSTJZNO0sJqFtrS3Rdk_aSBQNONXCYSOafIqCtjz1b4mZ4vA._Fj9x0E13gOT07yIeuhemw"
+        },
+    
+        "record": {
+            "CNAME": "https://hudson-liu.github.io/personal-website/"
```

Comment:
Invalid CNAME. CNAME records should be a single string which contains a valid hostname containing no protocols, file paths, and/or URL parameters.